DiagnosticsExtension/Models/ConnectionStringValidator/ConnectionStringValidationResult.cs

```diff
@@ -40,9 +40,20 @@ public enum ResultStatus
             MsiFailure,
             EmptyConnectionString,
             MalformedConnectionString,
-            UnknownError
+            UnknownError,
+            ManagedIdentityCredentialMissing,
+            FullyQualifiedNamespaceMissed,
+            SystemAssignedManagedIdentity,
```

Comment:
This and UserAssignedManagedIdentity don't make sense as result status.  They don't communicate any status or error.

Reply:
fixed